docs/guides/composing-messages.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: "Composing Messages"
+lang: en
+---
+
+# Composing Messages
+
+This section shows how to build Slack messages using **slack-api-client** library. If you're not familiar with **chat.postMessage** API yet, read [this page]({{ site.url | append: site.baseurl }}/guides/web-api-basics) before trying the samples here.
+
+Also, before jumping into Java code, we recommend developing a good understanding of the concept behind this SDK's Java interfaces. Reading the API document entitled [An overview of message composition](https://api.slack.com/messaging/composing) is the fastest way to know the basics about Slack message composition.
```

Comment:
"developing an understand of composing Slack messages. [Read the API documentation](https://api.slack.com/messaging/composing) for more information."